examples/contact/CMakeLists.txt

```diff
@@ -18,7 +18,12 @@ if(TRIBOL_FOUND)
         blt_add_executable(NAME        contact_${example_name}
                            SOURCES     ${filename}
                            OUTPUT_DIR  ${EXAMPLE_OUTPUT_DIRECTORY}
-                           DEPENDS_ON  serac_physics serac_mesh)
+                           DEPENDS_ON  serac_physics serac_mesh_utils)
+
+        # TODO Shorten runtime down to less than one minute
+        # blt_add_test(NAME            contact_${example_name}
+        #              COMMAND         contact_${example_name}
+        #              NUM_MPI_TASKS   4)
```

Comment:
ATM, these tests take too long to be in a regular CI job. Maybe we can have an option to tests examples nightly. Or we should somehow reduce the runtime of these examples similar to `--fast` in `cylinder.cpp`.

```
      Start 86: contact_beam_bending
86/93 Test #86: contact_beam_bending ....................   Passed   20.40 sec
      Start 87: contact_ironing
87/93 Test #87: contact_ironing .........................   Passed  277.27 sec
      Start 88: contact_sphere
88/93 Test #88: contact_sphere ..........................   Passed  294.65 sec
      Start 89: contact_twist
89/93 Test #89: contact_twist ...........................   Passed  143.91 sec
```

Reply:
@ebchin

Reply:
We can coarsen the meshes and/or just run them with 1 timestep for CI, but it'd be good to run them all the way once in a while.  In my experience, convergence issues don't appear until towards the end of the simulation and with more refined meshes.  I guess I'd vote for a nightly test for examples?